README.md

```diff
@@ -1,6 +1,6 @@
 # Artifactory and Xray Logging Analytics with Fluentd and Datadog
 
-The following describes how to configure Datadog to gather metrics from Artifactory and Xray through the use of FluentD.
+The following document describes how to configure Datadog to gather metrics from Artifactory and Xray through the use of FluentD.
 
 | version | artifactory | xray  | distribution | mission_control | pipelines |
```

Comment:
Remove this table since it does not have context as to why it is here. And if we need we should create a different section which talks about supported versions of different products and create a different page and link it at the bottom of this readme.

Reply:
Removing table as we're not maintaining versioning yet. As we start versioning our app, we can add this table separately